packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -89,60 +111,154 @@ export const prepareLiquidityPoolKit = (zone, zcf, USDC, tools) => {
      */
     (shareMint, node) => {
       const { brand: PoolShares } = shareMint.getIssuerRecord();
-      const proposalShapes = makeProposalShapes({ USDC, PoolShares });
-      const shareWorth = makeParity(makeDust(USDC), PoolShares);
+      const proposalShapes = makeProposalShapes({
+        USDC: usdc.brand,
+        PoolShares,
+      });
+      const shareWorth = makeParity(makeDust(usdc.brand), PoolShares);
       const { zcfSeat: poolSeat } = zcf.makeEmptySeatKit();
-      const shareWorthRecorderKit = tools.makeRecorderKit(node, RatioShape);
+      const { zcfSeat: contractSeat } = zcf.makeEmptySeatKit();
+      const poolMetricsRecorderKit = tools.makeRecorderKit(
+        node,
+        PoolMetricsShape,
+      );
+      /** used for `checkPoolBalance` invariant */
+      const outstandingLends = makeEmpty(usdc.brand);
+      const poolStats = /** @type {PoolStats} */ harden({
+        totalBorrows: makeEmpty(usdc.brand),
+        totalContractFees: makeEmpty(usdc.brand),
+        totalPoolFees: makeEmpty(usdc.brand),
+        totalRepays: makeEmpty(usdc.brand),
+      });
       return {
-        shareMint,
-        shareWorth,
+        contractSeat,
+        outstandingLends,
+        poolStats,
+        poolMetricsRecorderKit,
         poolSeat,
         PoolShares,
         proposalShapes,
-        shareWorthRecorderKit,
+        shareMint,
+        shareWorth,
       };
     },
     {
-      feeSink: {
+      borrower: {
+        lookupBalance() {
+          const { poolSeat } = this.state;
+          return poolSeat.getAmountAllocated('USDC', usdc.brand);
+        },
         /**
-         * @param {Amount<'nat'>} amount
-         * @param {Payment<'nat'>} payment
+         * @param {ZCFSeat} assetManagerSeat
```

Comment:
`assetManagerSeat` seems like a left-over.

```suggestion
         * @param {ZCFSeat} toSeat
```